web/server.js

```diff
@@ -45,6 +45,20 @@ module.exports = function(options) {
     });
   });
 
+  function skipCSRF(request, reply) {
+    return reply(true);
```

Comment:
Is this correct? From the docs:

```
'skip' - a function with the signature of function (request, reply) {}, which when provided, is called for every request. If the provided function returns true, validation and generation of crumb is skipped (defaults to false)
```

I think you just need to `return true` here?